pt/workshops.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page
+title: Workshops
+permalink: /pt/workshops/
+redirect_from: /workshops
```

Comment:
```suggestion
```

Same here as previously discussed regarding the about page: this would redirect readers of any language to the Portuguese Workshop page.